include/wx/unix/private/uilocale.h

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/unix/private/uilocale.h
+// Purpose:     Various locale-related helpers used under Unix systems only
+// Author:      Vadim Zeitlin
+// Created:     2021-08-14 (extracted from src/common/intl.cpp)
+// Copyright:   (c) 2021 Vadim Zeitlin <vadim@wxwidgets.org>
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_UNIX_PRIVATE_UILOCALE_H_
+#define _WX_UNIX_PRIVATE_UILOCALE_H_
+
+#include <wx/string.h>
```

Comment:
This results in an error with (repository-based) Xcode builds.

```suggestion
#include "wx/string.h"
```

Reply:
Thanks, I'll fix this, I have no idea why did I use `<>` here. But beyond this I think this file shouldn't be used under Mac at all, so I'll stop including it from `src/common/intl.cpp`.